### PR TITLE
HYPRE

### PR DESCRIPTION
**Pull request by minrk**
cc @xmjiao

possible additional maintainers:

- @dalcinl
- @basnijholt
- @ocefpaf

others?

### CODE REVIEW

File: recipes/hypre/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - toolchain
+  run:
+    - {{ mpi }}
+    - openblas 0.2.19
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libHYPRE.dylib  # [osx]
+    - test -f $PREFIX/lib/libHYPRE.so  # [linux]
+    - test -f $PREFIX/include/HYPRE.h
```
Comment (ericdill) [inline recipes/hypre/meta.yaml]: Add this to the test section:
```yaml
    - conda inspect linkages -p $PREFIX $PKG_NAME  # [linux]
    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
```
but that can be done in the feedstock too.

File: recipes/hypre/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ mpi }}
+    - openblas 0.2.19
+    - cmake
+    - toolchain
```
Comment (ericdill) [inline recipes/hypre/meta.yaml]: You could drop the explicit specification of `toolchain`. It's getting pulled as a build time dependency of `openblas`. It's not actually hurting anything, just unnecessarily specifying dependencies.

**Comment by ericdill** [comment]
Did you want to add any of those other maintainers? Given the silence from them, I'm assuming not?

**Comment by dalcinl** [comment]
@ericdill Feel free to list me as maintainer.

**Comment by ericdill** [comment]
Any of the proposed changes can be made in the feedstock. Thanks for the recipe @minrk , sorry for the review delay

**Pull request merged**